src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -292,32 +285,49 @@ protected void wrapUp(GHPullRequestCommitDetail[] page) {
         };
     }
 
-    @Preview
-    @Deprecated
-    public GHPullRequestReview createReview(String body, @CheckForNull GHPullRequestReviewState event,
-                                            GHPullRequestReviewComment... comments)
+    public GHPullRequestReview createReview(@Nullable String commitId, String body, GHPullRequestReviewEvent event,
+                                            GHPullRequestReviewComment... comments) throws IOException {
+        return createReview(commitId, body, event, Arrays.asList(comments));
+    }
+
+    public GHPullRequestReview createReview(@Nullable String commitId, String body, GHPullRequestReviewEvent event,
+                                            List<GHPullRequestReviewComment> comments) throws IOException {
+        List<DraftReviewComment> draftComments = toDraftReviewComments(comments);
+        return new Requester(root).method("POST")
+                .with("commit_id", commitId)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "commit_id" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)